### ISSUE TITLE
Rule blocked: CORERULES-9563 - invalid_duration operator gives error when checking None

### ISSUE THREAD

**Issue by ASL-rmarshall** [cdisc-org/cdisc-rules-engine#900]
- A .txt file with the JSON containing the full request: [Request.txt](https://github.com/user-attachments/files/17748414/Request.txt)

**Links to related JIRA Tickets**
- https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-9563
- https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-9564

**Rule Information**
- **Standard**: USDM
- **Rule ID**: DDF00061
- **Rule Description**: When specified, the lower limit of a timing window must be a non-negative duration in ISO 8601 format.

**Describe the bug**
When the value of the variable being check by the `invalid_duration` operator is missing (i.e., `NULL`, `None`, `NaN`, etc.), the rule produces the following error when trying to perform the `re.match` in `is_valid_duration` ([`business-rules/utils.py, Line70`](https://github.com/cdisc-org/business-rules/blob/e3066532a7f231bd23aa4bbaee1ca5a0d1e8b7cf/business_rules/utils.py#L70)):
> TypeError: expected string or bytes-like object

**Error returned from Rule Engine**
```json
{
    "TIMING": [
        {
            "executionStatus": "execution_error",
            "dataset": "Timing.xpt",
            "domain": "TIMING",
            "variables": [],
            "message": "rule execution error",
            "errors": [
                {
                    "dataset": "Timing.xpt",
                    "error": "An unknown exception has occurred",
                    "message": "expected string or bytes-like object"
                }
            ]
        }
    ]
}
```

**Expected behavior**
The `is_valid_duration` should not give an error when the input `duration` is empty (e.g., `NULL`, `None`, `NaN`, etc.). Instead, it should  return `False` in the same way that it returns `False` for an empty string (`""`).

This could be achieved just by converting non-string durations to strings - e.g., inserting something like the following at the beginning of the `is_valid_duration` function:

```python
    if not isinstance(duration, str):
        duration = str(duration)
```
